skaffold.yaml

```diff
@@ -81,64 +93,45 @@ deploy:
 
 profiles:
   - name: dev
-    deploy:
-      helm:
-        releases:
-          - name: im-group-whoami-{{ .ENVIRONMENT }}
-            namespace: whoami
-            createNamespace: true
-            remoteChart: im-group
-            repo: https://dhis2-sre.github.io/im-group
-            version: 0.3.0
-            setValueTemplates:
-              serviceAccount:
-                name: im-manager-{{ .ENVIRONMENT }}
-                namespace: instance-manager-{{ .ENVIRONMENT }}
+    patches:
+      - op: add
+        path: /deploy/helm/releases/-
+        value:
+          name: im-group-whoami-{{ .ENVIRONMENT }}
+          namespace: whoami
+          <<: *chart-config
+          setValueTemplates:
+            serviceAccount:
+              name: im-manager-{{ .ENVIRONMENT }}
+              namespace: instance-manager-{{ .CLASSIFICATION }}
 
   - name: prod
-    deploy:
-      helm:
-        releases:
-          - name: im-group-dev
-            namespace: dev
-            createNamespace: true
-            remoteChart: im-group
-            repo: https://dhis2-sre.github.io/im-group
-            version: 0.3.0
-            setValues:
-              serviceAccount:
-                name: im-manager-prod
-                namespace: instance-manager-prod
+    patches:
+      - op: add
+        path: /deploy/helm/releases/-
+        value:
+          name: im-group-dev
```

Comment:
Why do we use `dev` in name/namespace in prod?

Reply:
Maybe slightly ambiguous, but that's for the `prod` environment to be able to deploy to the `dev` namespace.

Reply:
Interesting 😅 why are we doing that?

Reply:
Because we want the `im-manager` deployed to the `instance-manager-prod` environment to be able to deploy to the `dev` namespace which is assigned to the `dev` group. If it's just the naming again, do you have any better alternatives?